src/platforms/common/data-management/sensitive-data/index.mdx

```diff
@@ -66,23 +66,15 @@ For more details, see [_Filtering Events_](../../configuration/filtering/).
 
 Instead of sending confidential information in plaintext, considering hashing it:
 
-```javascript
-Sentry.setTag("birthday", checksum_or_hash("08/12/1990"));
-```
+<PlatformContent includePath="sensitive-data/set-tag" />
```

Comment:
Would this show as non-existent in all other platforms now that doesn't have a sample?

Reply:
yep, that's why I'd like to tackle this https://github.com/getsentry/sentry-docs/pull/2491#issuecomment-705465385